tests/chainer_tests/links_tests/activation_tests/test_simplified_dropconnect.py

```diff
@@ -0,0 +1,184 @@
+import os
+import tempfile
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import gradient_check
+from chainer import links
+from chainer.serializers import npz
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+from chainer.utils import type_check
+
+
+def gen_mask(ratio, shape):
+    return numpy.random.rand(*shape) >= ratio * (1. / (1 - ratio))
+
+
+@testing.parameterize(*testing.product({
+    'in_shape': [(3,), (3, 2, 2)],
+    'x_dtype': [numpy.float16, numpy.float32, numpy.float64],
+    'W_dtype': [numpy.float16, numpy.float32, numpy.float64],
+}))
+class TestSimplifiedDropconnect(unittest.TestCase):
+
+    out_size = 2
+    ratio = 0.5
+
+    def setUp(self):
+        in_size = numpy.prod(self.in_shape)
+
+        self.link = links.SimplifiedDropconnect(
+            in_size, self.out_size,
+            initialW=chainer.initializers.Normal(1, self.W_dtype),
+            initial_bias=chainer.initializers.Normal(1, self.x_dtype))
+        self.link.cleargrads()
+        self.mask = gen_mask(self.ratio, self.link.W.shape)
```

Comment:
According to the specification written in the document, the shape of a mask should be `(n, M, N)`, where as this test actually creates a mask of `(M, N)`. So, please fix the shape of the mask.